mt7697/arduino/sample/sample.ino

```diff
@@ -0,0 +1,67 @@
+#include <LBLE.h>
+#include <LBLEPeriphral.h>
+
+#define BUTTON_PIN      (6)
+
+int nButtonChanged = 0;
+int bOn=0;
```

Comment:
Could you follow our coding style?
`int bOn = 0`

Reply:
Done with new commit.